### PR TITLE
fix nc branding

### PR DESCRIPTION
**Pull request by ChristophWurst**
cc @nextcloud/mail

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,42 @@
 # Mail
```
Comment (mar1u5) [inline README.md]: Lets add "Nextcloud" here again 😉

**Comment by mar1u5** [comment]
Could we please put every screenshot into the `screenshots` repo 😁

**Comment by jancborchardt** [comment]
Yup, Marius is right, the screenshots should not clutter up the app repos. :) I opened a pull request in the screenshots repo: https://github.com/nextcloud/screenshots/pull/20

@ChristophWurst can you remove the screenshot from here and amend the commit?

File: README.md (hunk 1 above)
Comment (ChristophWurst) [inline README.md]: sound like *Let's make Mail great again* :wink:

**Comment by ChristophWurst** [comment]
> Yup, Marius is right, the screenshots should not clutter up the app repos. :) I opened a pull request in the screenshots repo: nextcloud/screenshots#20
> 
> @ChristophWurst can you remove the screenshot from here and amend the commit?

I don't think that would clutter the repo, it's just one file. Furthermore I find it quite handy to have it in the mail git repo as you can then go back and forth between releases and see the appropriate screenshot.

**Comment by irgendwie** [comment]
>% grep -inr -P '(?<!@)owncloud'

>js/README.md:20:Generally, any code contributed to this repository should comply with the general [ownCloud coding style guidelines](https://doc.owncloud.org/server/9.1/developer_manual/general/codingguidelines.html).
[...]
lib/controller/aliasescontroller.php:5: * ownCloud - Mail
[...]

I think both lines should be adapted for Nextcloud! :-1:

**Comment by jancborchardt** [comment]
@irgendwie good find! :) Do you want to fix them and add a commit?

**Comment by mar1u5** [comment]
> I don't think that would clutter the repo, it's just one file. Furthermore I find it quite handy to have it in the mail git repo as you can then go back and forth between releases and see the appropriate screenshot.

😕 ... I think it does not make sense to store a screenshot in the repo **and** a separate repo. We have also no screenshots in calendar/contacts or server repo. But thats only my opinion. 
@ChristophWurst @jancborchardt

File: README.md (hunk 1 above)
Comment (mar1u5) [inline README.md]: FIRST THE CORE, THEN THE MAIL APP AND LAST BUT NOT LEAST **AMERICA** 😈😅 
donald-trump-trademarked-a-ronald-reagan-slogan-and-would-like-to-stop-other-republicans-from-using-it

File: js/views/aliases.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * ownCloud - Mail
+ * Mail
```
Comment (mar1u5) [inline js/views/aliases.js]: Is there a reason of why we are not simply replacing that with `Nextcloud - Mail` ?! 😁 Does not matter anyway - but yeah 😅

**Comment by irgendwie** [comment]
@jancborchardt Done - my grep filtering also shows some more occurrences of ownCloud, but I think we can ignore them for now.

**Comment by jancborchardt** [comment]
I tend to agree with @mar1u5 that we don’t really need to store the screenshots here as well. The screenshots repo has branches for releases as well and the look of the app doesn’t change that more often.

Anyway it’s your call @ChristophWurst. I adjusted the screenshot and compressed it.

**Comment by ChristophWurst** [comment]
thanks for the fixes @jancborchardt @irgendwie. Does this PR qualify for a :+1: now? LGTM is waiting for an approval comment :stuck_out_tongue:

**Comment by irgendwie** [comment]
Would say yes! LGTM :+1:

**Comment by jancborchardt** [comment]
By the way, you are all invited to the Nextcloud conference, come register at https://conf.nextcloud.com :)
(If you didn't register yet or didn't submit a lightning talk yet. ;) cc @nextcloud/mail @mar1u5 @zeugmatis

**Comment by zeugmatis** [comment]
Ah, I would totally go but I am in California!  Just north of SF.  Hope to try and sell NC here, but dropbox being free... is one tough competitor!  Maybe same time next year.  :-)

**Comment by jancborchardt** [comment]
@zeugmatis ooh cool! Maybe start a Nextcloud meetup there then? ;)

**Pull request merged**